SonarQube.MSBuild.Tasks/IsTestFileByName.cs

```diff
@@ -29,11 +29,6 @@ public sealed class IsTestFileByName : Task
         /// </summary>
         public const string TestRegExSettingId = "sonar.cs.msbuild.testProjectPattern";
 
-        // Workaround for the file locking issue: retry after a short period.
```

Comment:
The changes in this file are to move the common config-reading code out to a new class.